### DISCUSSION TITLE
Instantiate a simulation on the fly from different model in the same experiment

### DISCUSSION THREAD

**Discussion by agrignard** [gama-platform/gama#95] [Ideas]
**Is your request related to a problem? Please describe.**
I am not sure this is already possible in GAMA. If yes, what about having an exemple of it in the model library? If no here is the enhancement request I have in mind:

**Describe the solution you'd like**
I'd like to be able to run a "meta" experiment that would give me the possibility to instantiate any simulation on the fly by just pressing a key.

Example of application: in a demo space imagine you want to show an experiment running model A (e.g boids) and by simply pressing a button, or placing a specific piece on a tangible table, you would like to run another experiment from a model B (e.g ants).

Ideally the way I would see it in a pseudo-code style would be 
`````
import modelA.gaml
import modelB.gaml

...

experiment type:meta{
event["1"]  {create simulation model:modelA.gaml experiement:expA;};
event["2"]  {create simulation model:modelB.gaml experiement:expB;};
}
`````

**Additional context**
This mainly meant for demo purpose (e.g a demo room where you would like to show case different application without having to use the GUI)

**Comment by AlexisDrogoul** [comment]
Probably best answered by @hqnghi88 (which has implemented the already existing `run` and `start_simulation` statements). I will also move to the discussions if you do not mind .
